### PR TITLE
Add possibility to use URL for build.yaml

### PR DESCRIPTION
**Pull request by rshym**
PR is created to address https://github.com/xen-troops/moulin/issues/54

### CODE REVIEW

**Comment by rshym** [comment]
This PR as addressing https://github.com/xen-troops/moulin/issues/54

File: moulin/main.py [hunk 1]
```diff
@@ -53,6 +56,26 @@ def _prepre_shared_opts(description: str,
     return parser
 
 
+def _download_URL(url: str) -> str:
```
Comment (lorc) [inline moulin/main.py:59]: well, it not always downloads an URL, as seen in the code.

How about `_get_conf_file`?

Also, probably doc string should be updated accordingly.

**Comment by rshym** [comment]
Rebased to HEAD.
Function `_download_URL` is renamed to `_get_conf_file`, and doc string is updated.
File user-reference.rst is updated with info that file is downloaded only if it doesn't exist locally.
Version is bumped to 0.10.

File: moulin/main.py (hunk 1 above)
Comment (rshym) [reply moulin/main.py]: Done.

File: moulin/main.py [hunk 2]
```diff
@@ -53,6 +56,25 @@ def _prepre_shared_opts(description: str,
     return parser
 
 
+def _get_conf_file(url: str) -> str:
+    "Download file from URL, if it is not present. And return it's local name."
```
Comment (lorc) [inline moulin/main.py:60]: It is unclear what exactly "is not present". Is it file or URL?

How about "Try to determine config file location. Check if input string is URL and download it if output file is not already present. If input string is not an URL, then assume that this is a local file and return it as is."

File: moulin/main.py [hunk 3]
```diff
@@ -53,6 +56,25 @@ def _prepre_shared_opts(description: str,
     return parser
 
 
+def _get_conf_file(url: str) -> str:
```
Comment (lorc) [inline moulin/main.py:59]: Input string is not always URL. So parameter name is misleading.

File: moulin/main.py (hunk 2 above)
Comment (rshym) [reply moulin/main.py]: Thanks. I had a problem formulating that. I took your text as is.

File: moulin/main.py (hunk 3 above)
Comment (rshym) [reply moulin/main.py]: `url` replaced with `conf_location`.

**Comment by rshym** [comment]
Comments are addressed.

**Comment by lorc** [comment]
`Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by rshym** [comment]
RB is added.
Tested, see one of comments above.
Ready to be merged.

**Pull request merged**